README.md

```
@@ -1,628 +1,302 @@
 # philips-airctrl
 
-[![Python 3.8+](https://img.shields.io/badge/python-3.8+-blue.svg)](https://www.python.org/downloads/)
+[![Python 3.11+](https://img.shields.io/badge/python-3.11+-blue.svg)](https://www.python.org/downloads/)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
-[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
-[![Home Assistant](https://img.shields.io/badge/Home%20Assistant-Integration%20Ready-blue.svg)](https://www.home-assistant.io/)
+[![Code style: ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
+[![Tests](https://img.shields.io/badge/coverage-100%25-brightgreen.svg)]()
```

markdownlint-cli2 (0.20.0)

[warning] No empty links

(MD042, no-empty-links)